app/helpers/feedex_helper.rb

```diff
@@ -12,8 +11,22 @@ def total_responses_header(total_count:, from:, to:, results_limited:)
       "#{response_total} before #{to}"
     elsif total_count && total_count > 1 && !results_limited
       "All #{response_total}"
+    elsif is_done_page?(path)
+      "All responses"
     else
       response_total
     end
   end
+
+  def is_done_page?(path)
+    path.start_with?("done", "/done")
+  end
+
+
+  def set_response_total(total_count, results_limited, path)
```

Comment:
This isn't "setting" a response total, `get_response_total` would be more suitable.